contents/newsletter/failure-modes-of-a-team-lead.md

```diff
@@ -0,0 +1,143 @@
+
+Subheading for email: "and how to avoid them."
+/////
+
+Being a team lead is a strange job. You still code, but now the team's success is your responsibility too. It's two full-time jobs.
+
+And here's the tricky part: it's easy to slip into habits that slow your team down, kill motivation, and stall progress.
+
+Below are the failure modes we've seen our team leads fall into and how to avoid them. They're written for senior engineers, tech leads, staff engineers, EMs – basically anyone guiding a team, whether it's in their job title or not.
+
+{POLL}
+
+## 1. You're always the bottleneck
+
+![](https://media.licdn.com/dms/image/v2/D5610AQFPrI1pRqqgFA/image-shrink_800/image-shrink_800/0/1728154838097?e=2147483647&v=beta&t=H6AEd1utCpJBY8lhwhljSyUBW1LVO98uFZaCErRsJx4)
+<Caption>Source: [New Yorker](https://www.newyorker.com/cartoon/a28298)</Caption>
+
+### Symptom
+
+You insert yourself into every decision, review every PR, and plan every ticket. Shipping grinds to a halt as everyone waits for your decisions.
+
+### Why it happens
+
+In an effort to keep quality high and make the right calls, you start acting more like a PM than an engineer. It's the easiest way to feel in control. Unfortunately, it also turns you into the bottleneck.
+
+### How to fix it
+
+Dont treat engineers like children. They're capable of [making great product decisions](/newsletter/product-management-is-broken#2-engineers-make-product-decisions) and owning their success. Here are a few ways to empower them to do that:
+
+- **Work transparently in public Slack channels and shared docs.** It gives your team the visibility and context they need to make smart, independent decisions.
+- **Push decisions down.** When someone asks, "Should we do X or Y?"" reply with, "What do you think and why?". Ship their answer if it's 80% as good as yours.
+- **Don't be the sole point of communication between your team and others.** Instead, connect the right people together. Act as a bridge, not a gate.
+- **Define outcomes for your team, not tasks.** How they get there is up to them.
+  
+> 🏆 **You'll know you're successful if:** You can disappear for two weeks without affecting the quality and frequency of releases.
+
+## 2. You stopped writing code
+
+### Symptom
+
+Slack is your new IDE and your calendar is a tetris game of stand-ups, retros, and quick syncs. Your GitHub graph looks like an empty game of minesweeper.
+
+### Why it happens
+
+Meetings are the easiest way for you to keep an eye on things, so now you only manage the work instead of actually doing it. 
+
+Another factor is ego: all those invites make you feel important. They create an illusion of productivity, and nobody pings you afterward asking what you actually got done. Before you know it, you're stuck in a doom loop: more meetings → fewer commits → coding skills fade → meetings feel safer → repeat.
+
+![](https://res.cloudinary.com/dmukukwp6/image/upload/doomloo_b8a8302df4.png)
```

Comment:
"Meetings feel safer" feels like slightly the wrong wording. I'm wondering "Safer than what" -> (coding). Maybe "Meetings are your safe space" ?